backend/src/constants/env-vars-schema.js

```diff
@@ -0,0 +1,12 @@
+export default {
+  CLIENT_ID: /.+/,
+  CLIENT_ID_PRODUCTION: /.+/,
+  LOCAL_STORAGE_PATH: /.+/,
```

Comment:
`/[0-9]+-[a-z0-9]{32}\.apps\.googleusercontent\.com/` ?

That matches the ID's I'm using and ones I've previously used.

Reply:
Thanks :thumbsup: Isn't that a regex for `CLIENT_ID` and `CLIENT_ID_PRODUCTION` though? (The line you commented on is `LOCAL_STORAGE_PATH`)

Reply:
Yes